association_backend/association/domain/services/do_checkout.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+
+import stripe
+from association.domain.entities.stripe_entities import StripeCheckoutSessionInput
+from association.domain.entities.subscription_entities import (
+    Subscription,
+    SubscriptionState,
+    UserData,
+)
+from association.domain.exceptions import AlreadySubscribed
+from association.domain.repositories import AssociationRepository
+from association.settings import STRIPE_SUBSCRIPTION_API_SECRET
+
+logger = logging.getLogger(__name__)
+
+stripe.api_key = STRIPE_SUBSCRIPTION_API_SECRET  # 'sk_test_4eC39HqLyjWDarjtT1zdp7dc'
```

Comment:
stripe key here

Reply:
Now the api_key is passed at every Stripe request, the TOKEN is taken from the settings